### ISSUE TITLE
Can I mock a variadic function?

### ISSUE THREAD

**Issue by adks3489** [eranpeer/FakeIt#184]
I am trying to mock a variadic function. But it seems not be able to find the matching function.

I'm using Visual Studio 2015. The compile error message is:
```
'fakeit::Mock<Foo>::stub': no matching overloaded function found
```
Is there any way I can mock variadic functions? Thanks.

```
struct Foo
{
    virtual void bar(const char* format, ...)
    {
    }
};
Mock<Foo> mock;
Fake(Method(mock, bar));
```

**Comment by teramil** [comment]
@adks3489 have you got any answers about the issue? I have the same problem.

**Comment by adks3489** [comment]
@teramil Nope. I just make a fake implementation by self. I think it might not be able to do it.
